src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvSwitches/FeatureOverviewEnvSwitches.tsx

```diff
@@ -37,15 +70,14 @@ const FeatureOverviewEnvSwitches = () => {
     };
 
     return (
-        <div className={styles.container}>
-            <Tooltip
-                arrow
-                title="Environments can be switched off for a single toggle. Resulting in all calls towards the toggle to return false."
-            >
-                <h3 className={styles.header} data-loading>
-                    Feature toggle status
-                </h3>
-            </Tooltip>
+        <StyledContainer>
+            <StyledHeader data-loading>
+                Feature toggle status
+                <HelpIcon
+                    tooltip="Environments can be switched off for a single toggle. Resulting in all calls towards the toggle to return false."
```

Comment:
Looks like 1 sentence, unless "...toggle. It will result in all calls towards the toggle returning `false`."

Reply:
Ah nice catch. How about something like this?

> When a feature is switched off in an environment, it will always return false. When switched on, it will return true or false depending on its strategies.